src/modules/bids-asks/helpers/select-order-book.js

```diff
@@ -107,15 +109,19 @@ const selectAggregatePricePoints = memoize((outcomeID, side, orders, orderCancel
  * @return {Object} aggregateOrdersPerPrice
  */
 function reduceSharesCountByPrice(aggregateOrdersPerPrice, order) {
-  if (order && order.price && order.amount) {
-    const key = new BigNumber(order.price, 10).toFixed()
+  if (order && !isNaN(order.fullPrecisionPrice) && !isNaN(order.fullPrecisionAmount)) {
```

Comment:
Are you checking for `NaN` just as a sanity check (if so, good thinking!), or have you actually been getting orders back from augur-node populated with `NaN` values?  If the latter, that's something we need to fix / create an issue for.

Reply:
this was a sanity check, just incase ;)